packages/components/progress/__tests__/progress.test.tsx

```
@@ -61,6 +61,13 @@ describe('Progress.vue', () => {
     )
   })
 
+  test('circle stroke color uses css var by status', () => {
+    const wrapper = mount(() => <Progress type="circle" status="success" />)
+    expect(
+      wrapper.find('.el-progress-circle__path').attributes('stroke')
+    ).toBe('var(--el-color-success, #13ce66)')
```

ESLint

[error] Replace `⏎······wrapper.find('.el-progress-circle__path').attributes('stroke')⏎····).toBe('var(--el-color-success,·#13ce66)'` with `wrapper.find('.el-progress-circle__path').attributes('stroke')).toBe(⏎······'var(--el-color-success,·#13ce66)'⏎····`

(prettier/prettier)